### ISSUE TITLE
Sanitizing post body

### ISSUE THREAD

**Issue by JuulArthur** [magination/blog#53]
We need to remove noice when saving the posts like &nbsp;

**Comment by hanskhe** [comment]
Are these inserted by TinyMce? In that case, we should see if that has to be the case. If not, we can set up some sanitizer before a post is saved to the DB to replace them with normal spaces.

**Comment by JuulArthur** [comment]
These are set in by TinyMCE, but only if the user press enter or copy inn something with a new line. The thought is that we can remiove &nbsp; before we save it to the DB if it's the start of the post

**Comment by hanskhe** [comment]
But what if the user presses enter in the middle of a text to make a paragraph? Does TinyMCE insert something different? Don't we have to check the entire text?

PS: Use a "\" before &nbsp; or it will be rendered by GitHub as a space. I fixed it for you ;)

**Comment by JuulArthur** [comment]
Yes is should only be done if the post start with it. Should be allowed in the text
